wordpress-mobile/WordPress-Android
# Allow img url to be null in comment details

Currently, if the comment contains an `img` tag without `src` attribute, opening comment details will crash the app.

This PR fixes this, by allowing the image URL to be null. In this case, an error placeholder will be displayed.

[![Image from Gyazo](https://i.gyazo.com/b72dca9970a2fbd6bd5361794002905a.png)](https://gyazo.com/b72dca9970a2fbd6bd5361794002905a)

To test:
- On the web, write a comment that contains an `img` tag without a `src` attribute.
- Open that comment in the app.
- Confirm that the app does not crash.

## Regression Notes
1. Potential unintended areas of impact
None, as far as I can tell.

PR submission checklist:
